### DISCUSSION TITLE
Disk Space

### DISCUSSION THREAD

**Discussion by tpnsolutions** [postalserver/postal#1627] [Help with using Postal] [answered]
I read in the installation that it is recommended to have a minimum of 100GB. I am curious which directories tend to fill up the most as I'm working with DigitalOcean and planning on setting up one or more Volumes rather than building an insanely large droplet just to satisfy the disk space. Thanks in advance!

**Comment by willpower232** [comment, marked as answer]
Almost all of Postals disk space usage comes from the database so the fewer emails you send/keep, the less you'll need. With only transactional emails and default retention settings we have survived with 25GB since launch. On one occasion we have needed to reduce a retention setting but its entirely down to how you will be using Postal.

Alternatively, if you keep your database on a separate service (like AWS RDS), it will be even less of an issue.

**Reply by tpnsolutions** [reply]
Thanks, this should help with designing a scaling model as once we've done testing thing intend to at the least migrate MySQL and/or RabbitMQ to one more more dedicated environments to allow both scaling in disk space (via Volumes) and RAM/CPU.
